Source/ChocolateyGui/AsyncProcess/ProcessEx.cs

```diff
@@ -17,6 +17,7 @@ namespace ChocolateyGui.AsyncProcess
     using System.Diagnostics;
     using System.Threading;
     using System.Threading.Tasks;
+    using System.Windows.Documents;

```

Comment:
Ha ha, good catch.  Nothing, no idea how that got in there.  Rebased and pushed.